.github/workflows/stale.yml

```diff
@@ -14,4 +14,4 @@ jobs:
           # mark issues/PRs stale when they haven't seen activity in 180 days
           days-before-stale: 180
           # ignore checking issues with the following labels
-          exempt-issue-labels: "epic, discussion"
\ No newline at end of file
+          exempt-issue-labels: "epic, discussion"
```

Comment:
Let's actually nix these, per https://github.com/dbt-labs/dbt-core/issues/4888. Epics are no longer relevant (that was for our pre-Jira tool), and we're going to be using real GitHub discussions instead of issues labeled `discussion`